Source/WebCore/platform/graphics/gstreamer/GStreamerCommon.cpp

```diff
@@ -1399,4 +1399,13 @@ GRefPtr<GstBuffer> wrapSpanData(const std::span<const uint8_t>& span)
 
 #undef IS_GST_FULL_1_18
 
+#if !GST_CHECK_VERSION(1, 20, 0)
+GstBuffer* gst_buffer_new_memdup(gconstpointer data, gsize size)
+{
+    gpointer data2 = g_memdup2(data, size);
```

Comment:
Taking the style into account, this should be called something like `copiedData` or something similar.